CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ For older changes see the [archived Singularity change log](https://github.com/a
   <path>` option, to read OCI registry credentials from this custom file.
 - Add support for libsubid
 
+### Other Changes
+
```

Comment:
There's no need to add a new category here at this point.  It's true that for non-patch releases (like 1.4) we do make categories in the release announcement but we'll introduce that later.